Tests/SentryTests/Integrations/Feedback/SentryFeedbackTests.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+@testable import Sentry
+import XCTest
+
+class SentryFeedbackTests: XCTestCase {
+    func testSerialize() throws {
```

Comment:
So I have this test, but I'm looking for the best place to write another test (an examples of anything that might have previously been done similarly) that will inspect the actual envelope contents when calling `SentrySDK.captureFeedback`. I see a lot of tests that use `Invocations`, but that is not what I'm talking about. I want to make assertions on the actual contents of the envelope JSON. Anything like that?

Reply:
Yes, this sounds like a good path. Thanks!

Reply:
I am currently taking the request manager approach, which has a double benefit of not spamming the backend with every envelope sent when running these tests.

As a point of clarification, I would not call these end-to-end tests, but at most, integration tests. An E2E test IMO would exercise a mobile app with the SDK, and then validate on our web app that the right thing showed up as a customer would see it, by inspecting the DOM.

Reply:
see https://github.com/getsentry/sentry-cocoa/pull/4652